recipes/replxx/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+project(test_package)
+cmake_minimum_required(VERSION 2.8.11)
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
```

Comment:
I received some comments myself about removing this from my pr's.

Reply:
ok, just copy pasted from spdlog :"D